ups/meas_base.cfg

```diff
@@ -21,5 +21,5 @@ config = lsst.sconsUtils.Configuration(
     __file__,
     headers=["lsst/meas/base.h"],
     hasDoxygenInclude=False,
-    hasSwigFiles=True,
+    hasSwigFiles=False,
```

Comment:
@r-owen, are you sure we can safely delete this line? In `sconsUtils`, `hasSwigFiles` defaults to true.